.travis.yml

```diff
@@ -0,0 +1,52 @@
+language: julia
+
+os:
+  - linux
+  - osx
+  - freebsd
+
+julia:
+  - 1.0
+  - 1.4
+  - nightly
+
+env:
+  - DATADEPS_ALWAYS_ACCEPT=true
+
```

Comment:
Is [`DataDeps.jl`](https://github.com/oxinabox/DataDeps.jl) going to be used here?  If not, remove these lines
```suggestion
```